src/main/java/de/tum/in/www1/artemis/web/rest/lecture/AttachmentUnitResource.java

```diff
@@ -37,20 +37,27 @@ public class AttachmentUnitResource {
 
     private final AuthorizationCheckService authorizationCheckService;
 
-    public AttachmentUnitResource(AttachmentUnitRepository attachmentUnitRepository, LectureRepository lectureRepository, AuthorizationCheckService authorizationCheckService) {
+    private final GroupNotificationService groupNotificationService;
+
+    private final AttachmentUnitService attachmentUnitService;
+
+    public AttachmentUnitResource(AttachmentUnitRepository attachmentUnitRepository, LectureRepository lectureRepository, AuthorizationCheckService authorizationCheckService,
+            GroupNotificationService groupNotificationService, AttachmentUnitService attachmentUnitService) {
         this.attachmentUnitRepository = attachmentUnitRepository;
         this.lectureRepository = lectureRepository;
         this.authorizationCheckService = authorizationCheckService;
+        this.groupNotificationService = groupNotificationService;
+        this.attachmentUnitService = attachmentUnitService;
     }
 
     /**
-     * GET /lectures/:lectureId/attachment-units/:attachmentUnitId : gets the attachment unit with the specified id
+     * GET lectures/:lectureId/attachment-units/:attachmentUnitId : gets the attachment unit with the specified id
      *
      * @param attachmentUnitId the id of the attachmentUnit to retrieve
-     * @param lectureId the id of the lecture to which the unit belongs
+     * @param lectureId        the id of the lecture to which the unit belongs
      * @return the ResponseEntity with status 200 (OK) and with body the attachment unit, or with status 404 (Not Found)
      */
-    @GetMapping("/lectures/{lectureId}/attachment-units/{attachmentUnitId}")
+    @GetMapping("lectures/{lectureId}/attachment-units/{attachmentUnitId}")
```

Comment:
Why did you not add `value =` here when you did add it for the calls below?
Personally I'd leave it out everywhere

Reply:
I need it below because I also specify what MediaType this endpoint consumes. Therefore I also have to specify which of the parameters is the value.
When there is only one parameter, I don't need to specify it.